setup-production.adoc

```diff
@@ -31,217 +31,172 @@ $ cd taiga-docker/
-TAIGA_SUBPATH: "/taiga"
-----
+.**Email settings**
 
-**Service: taiga-events**
-[source, bash]
-----
-TAIGA_SECRET_KEY: "taiga-back-secret-key"
-----
+By default, emails will be printed in the standard output (`EMAIL_BACKEND=**console**`). If you have a SMTP service, change it to `EMAIL_BACKEND=**smtp**` and configure the rest of these variables:
 
-**Service: taiga-protected**
 [source, bash]
 ----
-SECRET_KEY: "taiga-back-secret-key"
-----
-
-`TAIGA_SECRET_KEY` or `SECRET_KEY` is the secret key of Taiga. Should be the same as this var in `taiga-back`, `taiga-events` and `taiga-protected`.
-`TAIGA_URL` in front service is where this Taiga instance should be served. It should be the same as `TAIGA_SITES_SCHEME://TAIGA_SITES_DOMAIN`.
-`TAIGA_WEBSOCKETS_URL` is used to connect to the events. This should have the same value as `wss://TAIGA_SITES_DOMAIN`, ie: wss://taiga.mycompany.com.
+EMAIL_BACKEND=console  # use an SMTP server or display the emails in the console (either "smtp" or "console")
+EMAIL_HOST=smtp.host.example.com  # SMTP server address
+EMAIL_PORT=587   # default SMTP port
+EMAIL_HOST_USER=user  # user to connect the SMTP server
+EMAIL_HOST_PASSWORD=password  # SMTP user's password
+EMAIL_DEFAULT_FROM=changeme@example.com  # email address for the automated emails
 
-.Session Settings
+# EMAIL_USE_TLS/EMAIL_USE_SSL are mutually exclusive (only set one of those to True)
+EMAIL_USE_TLS=True  # use TLS (secure) connection with the SMTP server
+EMAIL_USE_SSL=False  # use implicit TLS (secure) connection with the SMTP server
+----
 
-Taiga doesn't use session cookies in its API as it stateless. However, the Django Admin (`/admin/`) uses session cookie for authentication. By default, Taiga is configured to work behind HTTPS. If you're using HTTP (despite the strong recommendations against it), you'll need to configure the following environment variables so you can access the Admin:
+.**Queue manager settings**
+These variables are used to leave messages in the rabbitmq services.
 
-**Service: taiga-back**
 [source, bash]
 ----
-SESSION_COOKIE_SECURE: "False"
-CSRF_COOKIE_SECURE: "False"
+RABBITMQ_USER=taiga  # user to connect to RabbitMQ
+RABBITMQ_PASS=taiga  # RabbitMQ user's password
+RABBITMQ_VHOST=taiga  # RabbitMQ container name
+RABBITMQ_ERLANG_COOKIE=secret-erlang-cookie  # unique value shared by any connected instance of RabbitMQ
 ----
 
-More info about those variables can be found link:https://docs.djangoproject.com/en/3.1/ref/settings/#csrf-cookie-secure[here].
 
-.Email Settings
+.**Attachments settings**
 
-By default, email is configured with the *console* backend, which means that the emails will be shown in the stdout. If you have an smtp service, uncomment the "Email settings" section in `docker-compose.yml` and configure those environment variables:
+You can configure how long the attachments will be accessible by changing the token expiration timer. After that amount of seconds the token will expire.
```

Comment:
It seems to me that we are saying that the attachment will no longer be available after the expiration-age, and, if i recall properly it's more something like:

"the token will expire, but you can always get a new attachment url with an active token".

Reply:
I find it perfect. I'll add your comment.